doc/source/development/contributing.rst

```diff
@@ -157,9 +157,56 @@ Instead of manually setting up a development environment, you can use Docker to
+and download and install the `Linux kernel update package <https://docs.microsoft.com/pl-pl/windows/wsl/wsl2-kernel>`_.
+
+Download and install `Docker for Windows <https://docs.microsoft.com/pl-pl/windows/wsl/wsl2-kernel>`_.
+When prompted, ensure the `Enable WSL 2 Features` option is selected on the Configuration page. After installation docker will not start automatically. To start Docker Desktop, search for Docker, and select Docker Desktop in the search results.
+When the whale icon in the status bar stays steady, Docker Desktop is up-and-running, and is accessible from any terminal window.
+
+See  `official docker documentation <https://docs.docker.com/docker-for-windows/install-windows-home/>`_ for details.
+
+**Installing Docker on Windows Pro, Enterprise, or Education**
+
+First make sure that you have Windows 10 64-bit: Pro, Enterprise, or Education (Build 15063 or later). Ensure that Hyper-V feature is `enabled <https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v>`_.
+
+Download and install `Docker for Windows <https://docs.microsoft.com/pl-pl/windows/wsl/wsl2-kernel>`_.
+When prompted, ensure the `Enable Hyper-V Windows Features` option is selected on the Configuration page.
+If your admin account is different to your user account, you must add the user to the docker-users group. Run Computer Management as an administrator and navigate to `Local Users and Groups > Groups > docker-users`. Right-click to add the user to the group. Log out and log back in for the changes to take effect.
+Docker Desktop does not start automatically after installation. To start Docker Desktop, search for Docker, and select Docker Desktop in the search results.
+When the whale icon in the status bar stays steady, Docker Desktop is up-and-running, and is accessible from any terminal window.
+
+See `official docker documentation <https://docs.docker.com/docker-for-windows/install/>`__ for details.
+
+**Installing Docker on Unix**
+
+ See official docker documentation for detailed information on `installing <https://docs.docker.com/engine/install/>`_ Docker in various Linux distributions  and `post-installation steps <https://docs.docker.com/engine/install/linux-postinstall/>`_.
+
+**Installing Docker on Mac OS**
+
+See `official docker documentation <https://docs.docker.com/docker-for-mac/install/>`__ for details.
+
+**Test your Docker installation**
+
+Run the following command in your terminal::
+
+    docker run hello-world
+
+You should see a welcome message, your installation is complete! In case of problems refer to these pages with common issues on: `Windows <https://docs.docker.com/docker-for-windows/troubleshoot/>`_,
+`MacOS <https://docs.docker.com/docker-for-mac/troubleshoot/>`_.
+
+**Development inside Docker container**
```

Comment:
Could we put
```bash
docker build --tag pandas-dev:latest .
docker run -it --rm -v $PWD:/pandas-dev -w /pandas-dev pandas-dev:latest
```
somewhere in this section, and then note that, _as an alternative_, it's possible to launch a remote session within vs code using `.devcontainer.json`?